ui/src/store/modules/user.js

```diff
@@ -44,27 +44,19 @@ const generalMenuItems = [
 const state = {
   userData: {
     accountType: null,
-    email: null,
-    hasAcceptedPrivacyPolicy: false,
-    hasAcceptedTnC: false
+    email: null
   },
   token: null,
-  drawer: {
-    mini: false
-  },
+  hasAcceptedCookies: false,
+  hasAcceptedPrivacyPolicy: false,
+  hasAcceptedTnC: false,
   items: generalMenuItems
 };
 
 const mutations = {
   ...make.mutations(state),
   setUserData(state, data) {
     state.userData = { ...state.userData, ...data };
```

Comment:
Here, related to my previous comment, we only have setUserData in mutations and checkAccountTypeAndSetMenuItems in actions